packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -61,65 +48,235 @@ test('natMathHelpers', t => {
   );
   t.throws(
     () =>
-      coerce(
-        harden({ brand: { getAllegedName: () => 'somename' }, value: 4n }),
+      m.coerce(
+        {
+          brand: Far('otherBrand', {
+            getAllegedName: () => 'somename',
+            isMyIssuer: async () => false,
+            getDisplayInfo: () => ({}),
+          }),
+          value: 4n,
+        },
+        mockBrand,
       ),
     {
-      message: /The brand in the allegedAmount .* in 'coerce' didn't match the amountMath brand/,
+      message: /The brand in the allegedAmount .* in 'coerce' didn't match the specified brand/,
     },
     `coerce can't take the wrong brand`,
   );
   t.throws(
-    () => coerce(3),
-    { message: /The brand in allegedAmount .* is undefined/ },
+    () => m.coerce(3n, mockBrand),
+    {
+      message: `The brand in amount (a bigint) doesn't look like a brand. Did you pass a value rather than an amount?`,
+    },
```

Comment:
The message (below) should be 'coerce needs an amount', shouldn't it?

Reply:
changed